lib/model/Settings.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:openfoodfacts/utils/QueryType.dart';
+
+class Settings {
+  Settings._();
+
+  static String uriScheme = 'https';
+  static String uriProdHost = 'world.openfoodfacts.org';
+  static String uriTestHost = 'world.openfoodfacts.net';
+
+  static String uriProdHostRobotoff = 'robotoff.openfoodfacts.org';
+  static String uriTestHostRobotoff = 'robotoff.openfoodfacts.net';
+
+  static Uri getUri({
```

Comment:
good comments monsieurtanuki@.

For this one, I think static methods make sense here, specially because we may never want to create any instance of "Settings", since the app should only have 1 version of "settings". 

If there will never be more than one instance of "settings", I think it makes sense to use static methods rather than instance methods.

Reply:
@jasmeet0817 As I said before, I don't know how far this class is supposed to go.

That being said, I can still imagine cases where it's healthier to use rather `Settings.getInstance().getUri(...)` than `Settings.getUri(...)`.
E.g. if you want to run at the same time two queries: one with your standard settings ("your" current static version) and the other one with fallback settings (alternate server, fallback language like `'world'` or `'en'`, smaller page size). I don't say it's the most typical use case, but with the `static` syntax you just can't do that, and with the `getInstance()` syntax you can, and you can use the `copyWith` syntax.

Not sure what the standards on flutter/dart say about it, though.

Reply:
I think I understand what you mean but to achieve what you are writing about we would have to pass Settings to every method, this would again lead to much more comlexity which is probably not worth it,

Reply:
Not exactly, as `Settings.getInstance()` does not need to be passed as a parameter.
Maybe __later__ an optional `Settings` parameter, when the developer wants to override the "global" `Settings`.

Anyway, let's not spend too much time on this `getInstance` / `static` question for the moment; just finish your "static" version (in this PR and later). Then we'll see how important `Settings` are, and how relevant it would be to switch to the "instance" mode. Then, we publish to pub.dev.